worker/utils/database.py

```diff
@@ -9,10 +9,81 @@
     NotFoundError,
     ScheduledEmail,
     ScheduledEmailStatus,
-    to_scheduled_email,
 )
 
 
+class Db:
+    """Thin wrapper for DB queries. All methods are static and take a cursor
+    as a first argument, and every method executes a query and returns the cursor."""
+
+    @staticmethod
+    def select_scheduled_email(cursor: Cursor, id: UUID) -> Cursor:
+        return cursor.execute(
+            """
+            SELECT
+                id, created_at, last_updated_at, state, scheduled_at,
```

Comment:
I'll keep this as it is, because model changes should be additive to not break the worker.

Alternative approach is to use REST API instead of talking to the DB directly, which could be considered but out of scope of this PR.